src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSArgumentSetterConfig.java

```diff
@@ -56,7 +56,8 @@ public void validateProperties(FailureCollector collector) {
       }
     }
 
-    if (isServiceAccountJson()
+    Boolean isServiceAccountJson = isServiceAccountJson();
+    if (isServiceAccountJson != null && isServiceAccountJson
```

Comment:
This change is needed at Line#79 as well?

Reply:
in order for isServiceAccountJson to be null serviceAccountType needs to be null - and this can only happen if it's set as macro in validation stage which we are checking with ``canConnect`` on line 75